### ISSUE TITLE
Not working with Ticketmaster

### ISSUE THREAD

**Issue by Abolfazl** [ultrafunkamsterdam/undetected-chromedriver#351]
I ran the code:

```
import undetected_chromedriver.v2 as uc
driver = uc.Chrome()
with driver:
    driver.get('https://www.ticketmaster.com/')
```

and I am still getting the `As you were browsing something about your browser made us think you were a bot` message. Are there special options I need to add? My main goal is to receive the `reese84` cookie value.

**Comment by mew32jfd** [comment]
ever find a working chrome driver for this isssue

**Comment by FranciscoPalomares** [comment]
Update ?

**Comment by Abolfazl** [comment]
No, as soon as I use `undetected_chromedriver`, I get banned from ticketmaster for 24hours

**Comment by kpower7** [comment]
Any update on this? Also having the same issue :(

**Comment by Abolfazl** [comment]
I kept looking into this but could not resolve my problem. Had to abandon the project.

**Comment by kpower7** [comment]
> I kept looking into this but could not resolve my problem. Had to abandon the project.

Okay. If I can resolve I will let you know, but not hopeful.

Thank you for the response
